common/network/components/responder.c

```diff
@@ -6,21 +6,104 @@
  */
 
 #include "common/network/components/responder.h"
+#include "ciri/core.h"
+#include "ciri/node.h"
+#include "common/storage/sql/defs.h"
+#include "common/storage/storage.h"
 #include "utils/logger_helper.h"
 
 #define RESPONDER_COMPONENT_LOGGER_ID "responder_component"
 
+static bool get_transaction_to_reply(responder_state_t *const state,
+                                     hash_request_t *const request,
+                                     iota_transaction_t *const tx) {
+  if (state == NULL) {
```

Comment:
I think we can just use `||` ?

Reply:
I was getting it ready for https://github.com/iotaledger/entangled/issues/129 when they'll be different error codes